src/Domain/Services/DraftOrders.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Sets up all logic related to the Checkout Draft Orders service
+ *
+ * @package WooCommerce/Blocks
+ */
+
+namespace Automattic\WooCommerce\Blocks\Domain\Services;
+
+use Automattic\WooCommerce\Blocks\Domain\Package;
+
+/**
+ * Service class for adding DraftOrder functionality to WooCommerce core.
+ */
+class DraftOrders {
+
+	/**
+	 * Holds the Package instance
+	 *
+	 * @var Package
+	 */
+	private $package;
+
+	/**
+	 * Constructor
+	 *
+	 * @param Package $package An instance of the package class.
+	 */
+	public function __construct( Package $package ) {
+		$this->package = $package;
+	}
+
+	/**
+	 * Set all hooks related to adding Checkout Draft order functionality to Woo Core.
+	 */
+	public function init() {
+		if ( $this->package->is_feature_plugin_build() ) {
+			add_filter( 'wc_order_statuses', [ __CLASS__, 'register_draft_order_status' ] );
```

Comment:
Now moved, these methods no longer need to be static, or referenced with `__CLASS__`